### PR TITLE
Add total records counter

### PR DESCRIPTION
**Pull request by ArneZsng**
#### Features

🚀 Add total records counter

### CODE REVIEW

File: packages/ra-core/src/reducer/admin/resource/list/totalAll.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+export default resource => (previousState = 0, { type, payload, meta }) => {
+    if (!meta || meta.resource !== resource) {
+        return previousState;
+    }
+    if (type === CRUD_GET_ONE_SUCCESS) {
+        return previousState == 0 ? 1 : previousState;
+    }
+    if (type === CRUD_GET_LIST_SUCCESS) {
+        console.log('totalAll:', payload)
```
Comment (JakeDluhy) [inline packages/ra-core/src/reducer/admin/resource/list/totalAll.js]: Not sure your policy on committed console logs at this point, but I figured I’d point it out

**Review by JakeDluhy — APPROVED** [review]
Looks good! Just to make sure I understand this is so that when showing a filtered view the title still shows the total number of records right?

File: packages/ra-core/src/reducer/admin/resource/list/totalAll.js (hunk 1 above)
Comment (ArneZsng) [reply packages/ra-core/src/reducer/admin/resource/list/totalAll.js]: Oh yeah, that slipped through. Thanks for pointing it out.

**Review by tomknig — APPROVED** [review]

**Comment by tomknig** [comment]
@JakeDluhy exactly. The subtitle of the list view shows the number of all records of that resource in the database and the active quick-filter tab should show the number of records matching that filter.

**Pull request merged**